change/@fluentui-azure-themes-2020-10-09-16-58-10-rename-themes.json

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "patch",
+  "type": "prerelease",
```

Comment:
Do we need to change dependentChangeType here? 

Also if we only allow `prerelease` in master we should update the cherry-picking instructions to say to delete and re-generate the change files.

Reply:
yes we do need this. for example, `1.0.0-beta` with a new `patch` will be bumped to `1.0.0`. 
good call, for cherry-picking, the change files would require a bit more work.

Reply:
Makes sense. We may need to do another pass of fixing change types and package names in change files before publishing (and maybe again each day for a couple days) since there are likely open PRs with outdated change files.

Reply:
`yarn checkchange` will break after my change for change files which are not set to `prerelease` type.